scripts/jenkins/qa_openstack.sh

```diff
@@ -446,18 +446,45 @@ if [ -e /etc/tempest/tempest.conf ]; then
     fi
 
     pushd /var/lib/openstack-tempest-test/
+
+    # Handle OpenStack release specific blacklisting of known to fail tests
+    blacklist_args=""
+    case "${cloudsource}" in
+    (openstackrocky)
+        # TODO(fmccarthy): Remove once we have addressed issues causing
+        # failures for the neutron_tempest_plugin tests (SCRD-8681)
+        # If neutron_tempest_plugin is installed then blacklist known
+        # to fail tests
+        if rpm -q python-neutron-tempest-plugin > /dev/null 2>&1; then
+            _blacklist_file=$(mktemp /tmp/neutron_tempest_plugin_blacklist.XXXXXXXX)
```

Comment:
Nope, I'm happy to convert to a regular file in the tempest regular config directory... Is there a naming convention that should be followed for such files?

Creating a dynamically generated file under temp as a "safe" temp file using mktemp is just a good shell programming habit. And I was putting the dynamically generated file in /tmp because it is dynamically generated in the script.